### PR TITLE
ISPN-9577 fix deploy urls

### PR DESCRIPTION
**Pull request by pruivo**
https://issues.jboss.org/browse/ISPN-9577

### CODE REVIEW

**Comment by karesti** [comment]
@pruivo LGTM, could you just amend the two commits ?

File: build-configuration/pom.xml [hunk 1]
```diff
@@ -94,6 +94,13 @@
       <infinispan.module.slot>${infinispan.module.slot.prefix}-${infinispan.base.version}</infinispan.module.slot>
       <infinispan.core.schema.version>${infinispan.base.version}</infinispan.core.schema.version>
 
+      <!-- jboss repository urls -->
+      <jboss.releases.repo.id>jboss-releases-repository</jboss.releases.repo.id>
+      <jboss.releases.nexus.url>https://repository.jboss.org/nexus</jboss.releases.nexus.url>
+      <jboss.releases.repo.url>${jboss.releases.nexus.url}/service/local/staging/deploy/maven2/</jboss.releases.repo.url>
+      <jboss.snapshots.repo.id>jboss-snapshots-repository</jboss.snapshots.repo.id>
+      <jboss.snapshots.repo.url>${jboss.releases.nexus.url}/content/repositories/snapshots/</jboss.snapshots.repo.url>
```
Comment (karesti) [inline build-configuration/pom.xml]: releases ?

Comment (pruivo) [reply build-configuration/pom.xml]: it was the original name  :)

Comment (karesti) [reply build-configuration/pom.xml]: nevermind, looks good

**Comment by karesti** [comment]
seems good to me, just amend if you don't mind and I merge

**Comment by karesti** [comment]
@pruivo so can I integrate it now ?

**Comment by tristantarrant** [comment]
Merged

**Pull request merged**